src/kind.ts

```diff
@@ -55,12 +63,87 @@ export class KindConfig {
         return args;
     }
 
+    // returns the arguments to pass to `kind delete cluster`
+    deleteCommand(): string[] {
+        let args: string[] = ["delete", "cluster"];
+        if (this.name != "") {
+            args.push("--name", this.name);
+        }
+        if (this.kubeConfigFile != "") {
+            args.push("--kubeconfig", this.kubeConfigFile);
+        }
+        return args;
+    }
+
+    // returns the arguments to pass to `kind export logs`
+    exportLogsCommand(): string[] {
+        let args: string[] = ["export", "logs", this.kindLogsDir()];
+        if (this.name != "") {
+            args.push("--name", this.name);
+        }
+        return args;
+    }
+
+    kindLogsDir(): string {
+        const wd: string = process.env[`HOME`] || "";
+        let dir : string = "./.kind";
+        if (this.name != "") {
+            dir += `/${this.name}`;
+        }
+        dir += "/logs"
+        return path.join(wd, dir);
+    }
+
+    async executeKindCommand(command: string[]) {
+        console.log("Executing kind with args " + command.join(' ') );
+        await exec.exec("kind", command);
+    }
+
+    async installKind(): Promise<string> {
```

Comment:
(really unimportant nit): Both moving the body of a function _and_ renaming it makes it difficult to track the changes between revisions.

Reply:
I totally understand your point. Except by doing two commits or two pull_request, I don't see how it can be done. But it seems a lot 🤔